exercise1/IHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using RestSharp;
+using Retrofit.Net.Attributes.Methods;
+using Retrofit.Net.Attributes.Parameters;
+
+namespace exercise1
+{
+    public interface IMailChimpService
+    {
+        [Post("lists")]
+        RestResponse<bool> CreateList([Body] Object obj);
+
+        [Post("lists/{idList}/members")]
+        RestResponse<bool> AddEmail([Path("idList")] string id, [Body] object obj);
+
+        [Post("campaigns")]
+        RestResponse<bool> CreateCampaigns([Body] Object obj);
+
+        [Put("campaigns/{idCompaign}/content")]
+        RestResponse<bool> SetContent([Path("idCompaign")] string campaignId, [Body] object obj);
```

Comment:
Should change
- Make the variable name and the place-holder are same together. For example `campaignId` and `idCompaign`
- Specific type for `[Body]` and param name `obj` should be meaningful

Reply:
done